modules/carto/src/layers/carto-dynamic-tile-layer.js

```diff
@@ -86,7 +86,10 @@ export default class CartoDynamicTileLayer extends MVTLayer {
         Object.values(TILE_FORMATS).includes(formatTiles),
         `Invalid value for formatTiles: ${formatTiles}. Use value from TILE_FORMATS`
       );
-      url += `&${encodeParameter('formatTiles', formatTiles)}`;
+      // eslint-disable-next-line no-undef
+      const newUrl = new URL(url);
```

Comment:
Better use URLSearchParams instead of URL

Reply:
I think in this case URL is better because I'm actually modifying the URL